package.json

```diff
@@ -40,7 +40,7 @@
   "bugs": "https://github.com/nodeshift/openshift-rest-client/issues",
   "license": "Apache-2.0",
   "dependencies": {
-    "openshift-config-loader": "^0.4.0",
+    "kubernetes-client": "~6.11.0",
```

Comment:
Do you think `openshift-config-loader` is going the way of the dinosaur?

Reply:
it might be.  The kubernetes-client basically does the same thing, and it is a little better with the no SSL stuff too.